lib/home.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:connectuni/model/group.dart';
+import 'package:connectuni/model/group_card_view.dart';
+
+/**
+ * Landing/Home page that the User is introduced to when logging into their account.
+ */
+
+class HomePage extends StatefulWidget {
+  const HomePage({
+    super.key,
+  });
+
+  State<HomePage> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  int _selectedIndex = 0;
+
+  void _onItemTapped(int index) {
+    if (index != _selectedIndex) {
+      setState(() {
+        _selectedIndex = index;
+      });
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
Just a nitpick, you can remove the redundancy of const everywhere by doing:
`return const Scaffold( // code );` though I know its probably going to change down the road. Everything else LGTM! Nice work!